integrations/rwightman-timm/README.md

```diff
@@ -69,16 +69,15 @@ ViT recipes are not yet available on SparseZoo, but are in progress.
 
 Using a local recipe and checkpoint, pruning a model can be done by running the following command from within the root of this integration's folder
 ```bash
-python integrations/timm/train.py \
+python train.py \
   /PATH/TO/DATASET/imagenet/ \
-  --sparseml-recipe /PATH/TO/RECIPE/recipe.yaml \
-  --initial-checkpoint PATH/TO/CHECKPOINT/model.pth \
+  --recipe ../recipes/vit_base.85.recal.config.yaml \
```

Comment:
is the `..` necessary if we are in the directory with the train file?

Reply:
The train file is in the integration itself, whereas this is referencing the recipes folder on the sparseml. The ideas was that this example could be run out of the box.